sources/api/apiclient/Cargo.toml

```diff
@@ -17,7 +17,7 @@ futures.workspace = true
 futures-channel.workspace = true
 http.workspace = true
 httparse.workspace = true
-hyper = { workspace = true, features = ["client", "http1", "http2"] }
+hyper = { workspace = true, features = ["client", "http1", "http2", "tcp"] }
```

Comment:
I'm surprised that `http` doesn't imply `tcp`... but I guess Unix sockets and all that.